WordPressShared.podspec

```diff
@@ -21,7 +23,8 @@ Pod::Spec.new do |s|
   s.swift_version = '5.0'
 
   s.source        = { git: 'https://github.com/wordpress-mobile/WordPress-iOS-Shared.git', tag: s.version.to_s }
-  s.source_files  = 'Sources/WordPressShared/**/*.swift', 'Sources/WordPressSharedObjC/**/*.{h,m}'
+  s.source_files  = ['Sources/WordPressShared/**/*.swift'] \
+    + FileList['Sources/WordPressSharedObjC/**/*.{h,m}'].exclude('Sources/WordPressSharedObjC/include')
```

Comment:
TIL about Rake's [`FileList`](https://docs.ruby-lang.org/en/2.1.0/Rake/FileList.html) 🤓 

Do we need the `\` to go to new line?

```suggestion
  s.source_files  = ['Sources/WordPressShared/**/*.swift']
    + FileList['Sources/WordPressSharedObjC/**/*.{h,m}'].exclude('Sources/WordPressSharedObjC/include')
```